packages/web-pkg/src/composables/piniaStores/theme.ts

```diff
@@ -0,0 +1,139 @@
+import merge from 'deepmerge'
+import { defineStore } from 'pinia'
+import { ref, computed, unref } from 'vue'
+import { useLocalStorage, usePreferredDark } from '@vueuse/core'
+
+interface AppBanner {
+  title: string
+  publisher: string
+  additionalInformation: string
+  ctaText: string
+  icon: string
+  appScheme: string
+}
+
+interface CommonSection {
+  name: string
+  slogan: string
+  logo: string
+  urls: {
+    accessDeniedHelp: string
+    imprint: string
+    privacy: string
+  }
+}
+
+interface DesignTokens {
+  breakpoints?: Record<string, string>
+  colorPalette?: Record<string, string>
+  fontSizes?: Record<string, string>
+  sizes?: Record<string, string>
+  spacing?: Record<string, string>
+}
+
+interface LoginPage {
+  autoRedirect: boolean
+  backgroundImg: string
+}
+
+interface Logo {
+  topbar: string
+  favicon: string
+  login: string
+  notFound: string
+}
+
+interface ThemeDefaults {
+  appBanner?: AppBanner
+  common: CommonSection
+  designTokens: DesignTokens
+  loginPage: LoginPage
+  logo: Logo
+}
+
+interface WebTheme {
+  appBanner?: AppBanner
+  common?: CommonSection
+  designTokens: DesignTokens
+
+  isDark: boolean
+  name: string
+  loginPage?: LoginPage
+  logo?: Logo
+}
+
+interface WebThemeConfig {
+  defaults: ThemeDefaults
+  themes: WebTheme[]
+}
```

Comment:
Why no zod schema? :trollface:

Reply:
My plan was to parse it through zod in web-runtime/helpers/bootstrap.ts, but I guess reusing this schema throughout the place makes sense 😉 next step